### DISCUSSION TITLE
Print-optimized view of an Insights Dashboard

### DISCUSSION THREAD

**Discussion by olivermack** [directus/directus#21701] [Feature Requests]
### Summary

As an alternative to a native PDF export of a Dashboard (which was [already requested here](https://github.com/directus/directus/discussions/12926)) it would be nice to provide a Print-optimized version - e.g. solely based on CSS. This would enable a user to use pdf printers/exports.

Currently the print output of a dashboard's contents is just blank. Depending on the orientation I can manage to see the app shell (nav etc.) but actually the most important main content is just not visible.

### Motivation

Consider a setup which provides access to certain dashboards for external users where the dashboards are a vital part of the app design. The external user wants to persist the output of the dashboard in some artifact.

### Detailed Design

When using the browser's native Print feature the page should include the elements of the dashboard.

### Must Have:
- all elements of the dashboard are visible in the print mode

### Drawbacks

-

### Alternatives

Provide a native PDF export (see https://github.com/directus/directus/discussions/12926)

### Adoption Strategy

Just a general improvement which would probably not impact any existing installation negatively

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
